submissions/oparindeJolaoluwa/database/init.sql

```diff
@@ -0,0 +1,73 @@
+-- create a database for foodbag
+
+CREATE DATABASE foodbag;
+
+-- show the database
+
+SHOW DATABASES;
+
+-- select the foodbag database
+
+USE foodbag;
+
+-- Table 1 Food Bag 
+
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    type VARCHAR(100) NOT NULL,
+	fast_food VARCHAR(100) NOT NULL,
+    veg_fruits VARCHAR(100) NOT NULL,
+    drinks_cock VARCHAR(100) NOT NULL,
+    resturants VARCHAR(100) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- Table 2 Vendor Food
+
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(250) NOT NULL,
+    amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (food_id) REFERENCES food (id)
```

Comment:
Note that the admin_id is a foreign key in this table, so you ought to reference the admin table here.